wordpress-mobile/WordPress-Android
# Show selfhosted user info in login epilogue

Fixes #6379 

This PR depends on the FluxC PR https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/633 and that one should be merged first.

Enables the login epilogue header section for selfhosted sites. The section is populated with the site user's Gravatar, Display Name and Username.

To test:
* With the app data cleared, launch the app and login to a selfhosted site.
* On the epilogue screen, notice the Gravatar, user's Display Name (can be the same as username, depends on site's WP settings) and username.
